manager/runner.go

```diff
@@ -513,6 +534,12 @@ func (r *Runner) Run() error {
 				// Record that at least one template was rendered.
 				renderedAny = true
 
+				// Send the signal that the template got rendered
+				select {
+				case r.renderedCh <- tmpl.Path:
+				default:
```

Comment:
Yeah I think the access pattern will be select on channel, get map of updates. I am going to change the channel only to be one buffered and make it a chan struct{} so that is the only valid way to access it. That will solve this problem